DataFormats/Phase2HLTPFTaus/BuildFile.xml

```diff
@@ -0,0 +1,8 @@
+<use   name="DataFormats/Candidate"/>
```

Comment:
Do we really need a new package for a single class ?
Can it go in an existing one, for example `DataFormats/TauReco` or `DataFormats/HLTReco` ?